asv_bench/benchmarks/utils.py

```diff
@@ -211,20 +294,57 @@ def generate_dataframe(
     return df
 
 
-def random_string():
-    return str(uuid.uuid1())
+def random_string() -> str:
+    """
+    Create 36 character random strings using functionality from `uuid` module.
```

Comment:
```suggestion
    Create 36-character random strings.
```
Docstring does not have to explain _how_ stuff is done unless it is critical for the function's role.

Based on function name I think we don't care that these numbers are actually UUID-s. If that isn't true (i.e. if we rely somewhere on them being UUIDs) we should rename the function.